src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.search-label {
+  display: flex;
+  position: relative;
+  margin-top: 20px;
+}
+
+.input {
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.small {
```

Comment:
what is small? It is now not clear to me. Let's use clear naming for our classnames